index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="/css/reset.css" />
+    <link rel="stylesheet " href="/css/header.css" />
+    <link rel="stylesheet " href="/css/main.css" />
+    <link rel="stylesheet " href="/css/footer.css" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <h1>
+          <a id="logo" href="/" rel="home" aria-label="판다마켓 홈"> </a>
+        </h1>
+        <a href="/login.html" id="login-button">로그인</a>
+      </div>
+    </header>
+    <main>
+      <section class="top-bottom-section part-1">
+        <div>
+          <h2 class="sub-title">
+            일상의 모든 물건을 <br />
+            거래해 보세요
+          </h2>
+          <a href="/items.html">구경하러 가기</a>
+        </div>
+        <div>
+          <img src="imgs/Img_home_top.png" alt="언덕위 판다" />
+        </div>
+      </section>
+      <section class="main-section part-2">
+        <div>
+          <img src="imgs/Img_home_01.png" alt="판다 이미지 1" />
+        </div>
+        <div>
+          <div class="highlight">Hot itme</div>
+          <h2 class="sub-title">
+            인기 상품을 <br />
```

Comment:
❗️ 수정요청
지금 디자인에서 괜찮을 것 같지만 공백은 의도치 않은 동작을 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

```suggestion
            인기 상품을<br />
```